### PR TITLE
added NoResultsPage

### PR DESCRIPTION
**Pull request by ryoizawa**
Added NoResultsPage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
```
Comment (cherylli) [inline package.json:12]: Are these new dependencies used anywhere?

**Review by cherylli — COMMENTED** [review]
Just a question about the new dependencies otherwise look good

File: package.json (hunk 1 above)
Comment (ryoizawa) [reply package.json]: Sorry, I've added this but never be used and forgot to delete it.

**Comment by ryoizawa** [comment]
I've deleted the dependency there.

**Pull request merged**